pkgs/applications/office/jabref/default.nix

```diff
@@ -41,6 +41,9 @@ stdenv.mkDerivation rec {
     pname = "${pname}-deps";
     inherit src version;
 
+    postPatch = ''
+      sed -i -e 's/VERSION_18/VERSION_17/g' build.gradle
+    '';
```

Comment:
Testing locally, the patch here was unnecessary.

Reply:
Removed, thanks!

Reply:
It is still in the diff

Reply:
This is another patch which is removed. The remained patch is needed.